<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="assets/ico/favicon.png">

    <title>Profile - Takanori Hasegawa, Ph.D</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/css/bootstrap.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="assets/css/main.css" rel="stylesheet">
    
	<link rel="stylesheet" href="assets/css/font-awesome.min.css">
    
    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
    
    <script src="assets/js/jquery.min.js"></script>
    <script type="text/javascript" src="assets/js/smoothscroll.js"></script> 
    <script src="assets/js/Chart.js"></script>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="assets/js/html5shiv.js"></script>
      <script src="assets/js/respond.min.js"></script>
    <![endif]-->
  </head>

  <body data-spy="scroll" data-offset="0" data-target="#nav">
    <div id="section-topbar">
      <div id="topbar-inner">
	<div class="container">
	  <div class="row">
	    <div class="dropdown">
	      <ul id="nav" class="nav">
		<li class="menu-item"><a class="smoothScroll" href="#about" title="About"><i class="icon-user"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#resume" title="Resume"><i class="icon-file"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#current_work" title="Works"><i class="icon-briefcase"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#pub" title="Publications"><i class="icon-pencil"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#pic" title="Pictures"><i class="icon-camera"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#contact" title="Contact"><i class="icon-envelope"></i></a></li>
	      </ul><!--/ uL#nav -->
	    </div><!-- /.dropdown -->
	    
	    <div class="clear"></div>
	  </div><!--/.row -->
	</div><!--/.container -->
	
	<div class="clear"></div>
      </div><!--/ #topbar-inner -->
    </div><!--/ #section-topbar -->
    
    <div id="headerwrap">
      <div class="container">
	<div class="row centered">
	  <div class="col-lg-12">
	    <h1>Takanori Hasegawa, Ph.D</h1>
	    <h3>Junior Associate Professor at Tokyo Medical and Dental University</h3>
	    <h3>Co-founder and Collaborative Researcher at miup Inc.</h3>
	    <h3>Business and Technology Advisor at CAREER CO., LTD. and Scala Group.</h3>
	  </div><!--/.col-lg-12 -->
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/.#headerwrap -->    
    
    <section id="about" name="about"></section>
    <div id="intro">
      <div class="container">
	<div class="row">
	  
	  <div class="col-lg-2 col-lg-offset-1">
	    <h5>ABOUT</h5>
	  </div>
	  <div class="col-lg-6">
	      He obtained his Ph.D. on statistical time series analysis and computational biology from the Graduate School of Informatics, Kyoto University. During his doctoral studies, he also co-founded a start-up called "CRUNCHERS Inc.," to bring Information Science together with Art. At CRUNCHERS Inc., he developed a novel evaluation system centered on natural language processing technology and a box-office revenue forecasting system for movies (CRUNCH CINEMA). The start-up transferred its business in January 2015. <br/><br/>

	      After his Ph.D., he became an Assistant Professor at Tohoku Medical Megabank Organization, Tohoku University, where he focused on genome wide association study (GWAS) especially for rapid analysis of rare mutations. He also became a technical advisor to a team that was developing an AI-based medical disease inference system and an influenza-infection prediction system based on time-series data. <br/><br/>

	      He was appointed to an Assistant Professor at the University of Tokyo in 2015, where he has four key focus areas. First, he is involved in the development of a novel statistical methodology for the inference of the internal states of time series data and the prediction of their future states. Second, he focuses on cancer-immunogenomics and the development of their analysis tools in the International Cancer Genome Consortium. Third, he handles the stratification analysis of GWAS using cohort data, and lastly, he is involved in the predictive analysis of time-series health check-up data integrating blood test values, social status, lifestyle habits, genomic data, and so on. <br/><br/>

	      He was appointed to his current position in 2020, where he continues to his previous four areas and started to promote collaborative researches and establish analysis environments/education in Tokyo Medical and Dental University. He also has experience serving on committees and evaluation boards at various ministries and agencies. He has extensive knowledge of labor laws and dispatch law operations at medical sites. 
		</p>
	  </div>
	  <!--div class="col-lg-3">
	      <p><a href="#"><i class="icon-file"></i></a> <sm>DOWNLOAD PDF</sm></p>
	  </div!--->
	  
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/ #intro -->

    
    <section id="resume" name="resume"></section>
    <!--EDUCATION DESCRIPTION -->
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>EDUCATION</h5>
	</div>
	<div class="col-lg-6">
	  <p><t>Doctor of Informatics</t><br/>
	    Department of  Intelligence Science and Technology<br/>
	    Graduate School of Informatics, Kyoto University<br/>
	    (Bioinformatics Center, Institute for Chemical Research, Kyoto University)<br/>
	    <br/>
	    <i>Doctoral Program</i><br/>
	    Thesis Title: Reconstructing Biological Systems Incorporating Multi-Source Biological Data via Data Assimilation Techniques<br/>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2012 to Jan. 2015 (Completion in shortened years)</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Master of Information Science and Technology</t><br/>
	    Department of Computer Science <br/>
	    Graduate School of Information Science, The University of Tokyo<br/>
	    (Human Genome Center, Institute of Medical Science, The University of Tokyo)<br/>
	    <br/>
	    <i>Master's Program</i><br/>
	    Thesis Title: Intracellular Systems Analysis by Assimilating Large Scale Biological Data and Pathway Simulation<br/>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2010 to Mar. 2012</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Bachelor of Engineering</t><br/>
	    Facalty of Science and Engineering, Waseda University<br/>
	    <br/>
	    <i>Baccalaureate Degree Program</i><br/>
	    Thesis Title: 遺伝子ネットワーク推定とデータ同化による細胞内ネットワークシミュレーションモデルの構築</br>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2006 to Mar. 2010</sm></p>
	</div>
	
      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->
    
    
    <!--WORK DESCRIPTION -->
    <section id="current_work" name="current_work"></section>
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>CURRENT WORK</h5>
	</div>
	
	<div class="col-lg-6">
	  <p><t>Junior Associate Professor</t><br/>
	    <a href="http://www.tmd.ac.jp/cmn/dsc/index.html">Department of Integrated Analytics, M&D Data Science Center, Tokyo Medical Dental University</a><br/>
	  </p>
	  <p>My research topics are analyzing human genome, transcriptome, epigenome, metagenome and clinical data for personalized and preventive medicine through modeling, prediction and inference of disease and health systems. For example, we have been undertaking immunological cancer genome and metagenome analysis based on statistical and machine learning methods. <br/><br/>

		Especially, in the field of statistical science, my speciality is computational statistics and stochastic simulation, such as data assimilation and have published many research papers. <br/><br/>
					
		In addition, I'm promoting and establishing collaborative researches with medical and informatics units in TMDU and education for medical informatics. 
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>April 2020 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Co-founder / Collaborative Researcher</t><br/>
	    <a href="http://miup.jp/">miup</a><br/>
	  </p>
	  <p>
	    miup Inc. is trying to establish "Integrated Healthcare Eco-System" using IoT technology and clinical A.I. system starting from developing countries to developed countries. Firstly, we have launched <a href="http://my-sheba.com/">a home delivery health-care service in Bangladesh</a>. <br/><br/>

	    My role is to design business strategy, make financial indication and capital policy, raise funds, and manage our team. <br/><br/>

	    Our system has a highly complicated statistical structure to enhance the quality of the inference, estimate the confidence level of the conclusions, and predict the future risks through Bayesian statistics and machine learning. </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>September 2015 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Business and Technology Advisor</t><br/>
	    <a href="https://www.careergift.co.jp/">CAREER CO., LTD</a><br/>
	  </p>
	  <p>Detail is Confidential: Development of new business, and Operation and Production optimization.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>July 2019 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Business and Technology Advisor</t><br/>
	    <a href="https://scalagrp.jp/">Scala Group</a><br/>
	  </p>
	  <p>Detail is Confidential: Development of new business and Evaluation of start-up companies.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Aug. 2019 - Current</sm></p>
	</div>

      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->

    <!--WORK DESCRIPTION -->
    <section id="past_work" name="past_work"></section>
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>PAST WORK</h5>
	</div>

	<div class="col-lg-6">
	  <p><t>Assistant Professor</t><br/>
	    <a href="http://hic-imsut.jp/">Health Intelligence Center, The Institute of Medical Science, The University of Tokyo</a><br/>
	  </p>
	  <p>My research topics are analyzing human genome, transcriptome, epigenome, metagenome and clinical data for personalized and preventive medicine through modeling, prediction and inference of disease and health systems. For example, we have been undertaking immunological cancer genome and metagenome analysis based on statistical and machine learning methods. <br/><br/>
	    
	    Especially, in the field of statistical science, my speciality is computational statistics and stochastic simulation, such as data assimilation and have published many research papers.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>October 2015 - March 2020</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Technology Advisor</t><br/>
	    <a href="https://www.panasonic.com/global/home.html">Panasonic Inc.</a><br/>
	  </p>
	  <p>Detail is Confidential: Production and experiment optimization of an image processing devision based on Bayesian framework.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Apr 2017 - Mar. 2019</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Technology Advisor</t><br/>
	    <a href="http://medley.life/">Medley, Inc.</a><br/>
	  </p>
	  <p>We developed an automated consulting system termed <a href="http://medley.life/symptoms">Mogul</a> for medical treatment. This system is a type of artificial intelligence and based on a simple Bayesian inference procedure (Its concept is similar to the one in miup Inc. but this version is a little bit older). Also, a pandemic prediction system for the degree of future influenza epidemic has been released in the website. </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>February  2015 - September 2017 </sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Assistant Professor</t><br/>
	    <a href="http://www.megabank.tohoku.ac.jp/english/">Tohoku Medical Megaban Organization, Tohoku University</a><br/>
	  </p>
	  <p>The main topic was genome wide association study and to develop novel strategies for analyzing rare variants association test. Our developed method is highly efficient and available to WGS data within a reasonable time. In addition, we proposed a novel statistical method termed kernel Bayes' approximate Bayesian computation filtering for estimation of parameter values in stochastic simulation models.  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>February  2015 - September 2015</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Co-founder / CTO (Technology Advisor, Feb. 2015-)</t><br/>
	    <a href="http://www.crunchers.jp/">CRUNCHERS Inc. </a><br/>
	  </p>
	  <p>We developed automated evaluation systems for predicting the profit of books and cinema. The former system was integrated to Medley, Inc. as <a href="http://crmg.me/">CRUNCH MAGAZINE</a>. For the latter system, I still have advised the statistical perspective of applying the developed simulation system (CRUNCH CINEMA) to the collaborative companies. In addition, since we're collaborating with major studios and major entertainment companies in JAPAN and USA, we have extended and developed CRUNCH CINEMA for the application of general purpose.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>July 2013 - September 2015</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Research Fellow</t><br/>
	    Japan Society for The Promotion of Science<br/>
	  </p>
	  <p>The research topic was Reconstructing Biological Systems Incorporating Multi-Source Biological Data via Data Assimilation Techniques.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>April 2012 - January 2015</sm></p>
	</div>
	
      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->
    
    
    <!--AWARDS DESCRIPTION -->
    <section id="pub" name="pub"></section>
    <div class="container desc">
      <div class="row">
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>PUBLICATIONS</h5>
	</div>
	<div class="col-lg-9">
	  <p><t>Journal Paper</t><br/>
	    [1,F1] <u>T. Hasegawa</u>, R. Yamaguchi, M. Nagasaki, S. Imoto and S. Miyano. Comprehensive pharmacogenomic pathway screening by data assimilation. Proc. 7th International Symposium on Bioinformatics Research and Applications, volume 6674 of Lecture Notes in Computer Science, pp.160-171. Springer Berlin Heidelberg, 2011.<br/>
	    [2,F2] <u>T. Hasegawa</u>, M. Nagasaki, R. Yamaguchi, S. Imoto and S. Miyano. An efficient method of exploring simulation models by assimilating literature and biological observational data, BioSystems, vol.121, pp.54-66, 2014.<br/>
	    [3,F3] <u>T. Hasegawa</u>, R. Yamaguchi, M. Nagasaki, S. Miyano and S. Imoto. Inference of gene regulatory networks incorporating multi-Source biological knowledge via a state space model with L1 regularization. PLOS ONE, vol.9(8), e105942, 2014.<br/> 
	    [4,F4] <u>T. Hasegawa</u>, T. Mori, R. Yamaguchi, S. Imoto, S. Miyano and T. Akutsu. An efficient data assimilation schema for restoration and extension of gene regulatory networks using time-course observation data. Journal of Computational Biology, vol.21(11), pp.785-798, 2014.<br/>
	    [5,F5] <u>T. Hasegawa</u>, T. Mori, R. Yamaguchi, T. Shimamura, S. Miyano, S. Imoto and T. Akutsu. Genomic data assimilation using a higher moment filtering technique for restoration of gene regulatory networks. BMC Systems Biology, vol.9(14), pp.1-14, 2015.<br/>
	    [6,F6] <u>T. Hasegawa</u>, A. Niida, T. Mori, T. Shimamura, R. Yamaguchi, S. Miyano, T. Akutsu and S. Imoto. A likelihood-free filtering method via approximate Bayesian computation in evaluating biological simulation models. Computational Statistics and Data Analysis, vol.94, pp.63-74, 2016.<br/>
	    [7,F7] J. Kurashige, <u>T. Hasegawa (Equal First)</u>, et al. Integrated molecular profiling of human gastric cancer identifies DDR2 as a potential regulator of peritoneal dissemination. Scientific Reports, vol.6, p.22371, 2016.<br/>
	    [8] E. Ayada, <u>T. Hasegawa</u>, A. Niida, S. Miyano and S. Imoto. Binary contingency table Method for analyzing gene mutation in cancer genome. International Journal of Bioinformatics Research and Applications (11th The International Symposium on Bioinformatics Research and Applications), vol.12(3), pp.211-226, 2016.<br/>
	    [9] K. Kojima, Y. Kawai, N. Nariai, T. Mimori, <u>T. Hasegawa</u> and M. Nagasaki. Short tandem repeat number estimation from paired-end sequence reads by considering unobserved genealogy of multiple individuals. BMC Genomics (11th The International Symposium on Bioinformatics Research and Applications), vol.17(Suppl. 5), pp.465-476, 2016.<br/>
	    [10] S. Koshiba, Y. Yamaguchi, K. Kojima, <u>T. Hasegawa</u>, et al. The structural origin of metabolic quantitative diversity. Scientific Reports, vol.6, pp.31463, 2016.<br/>
	    [11,F8] <u>T. Hasegawa</u>, K. Kojima, Y. Kawai, K. Misawa, T. Mimori and M. Nagasaki. AP-SKAT: highly-efficient genome-wide rare variant association test. BMC Genomics, vol.17(1), pp.1-8, 2016.<br/>
	    [12,F9] <u>T. Hasegawa</u>, S. Hayashi, E. Shimizu, S. Mizuno, R. Yamaguchi, S. Miyano, H. Nakagawa, and S. Imoto. An in silico automated pipeline to identify tumor specic neoantigens from whole genome and exome sequencing data. Proc. 12th International Symposium on Bioinformatics Research and Applications, 2016.<br/>
	    [13] T. Morita, A. Rahman, <u>T. Hasegawa</u>, A. Ozaki, and T. Tanimoto. The Potential Possibility of Symptom Checker. International Journal of Health Policy and Management, vol.6(10), pp.615–616, 2017. <br/>
	    [14] T. Saito, ..., <u>T. Hasegawa</u>, et al. A temporal shift of the evolutionary principle shaping intratumor heterogeneity in colorectal cancer. Nature Communications, vol.9(1), p.2884, 2018. <br/>
	    [15,F10] <u>T. Hasegawa</u>, K. Kojima, Y. Kawai, and M. Nagasaki. Time-series filtering for replicated observations via a kernel approximate Bayesian computation. IEEE Transactions on Signal Processing, vol.66(23), pp.6148-6161, 2018. <br/>
	    [16] A. Niida, <u>T. Hasegawa</u>, S. Miyano. Sensitivity analysis of agent-based simulation utilizing massively parallel computation and interactive data visualization. PLOS ONE, vol.14(3), e0210678, 2019. <br/>
	    [17] The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium. Pan-cancer analysis of whole genomes. vol.578, pp.82–93, 2020. <br/>
	    [18] N. Sato, M. Kakuta, E. Uchino, <u>T. Hasegawa</u>, R. Kojima, W. Kobayashi, K. Sawada, Y. Tamura, I. Tokuda, S. Imoto, S. Nakaji, K. Murashita, M. Yanagita and Y. Okuno. The relationship between cigarette smoking and the tongue microbiome in an East Asian population. Journal of Oral Microbiology, vol.12(1), p.1742527, 2020.<br/>
	    [19,F11] <u>T. Hasegawa</u>, R. Yamaguchi, A. Niida, S. Miyano, S. Imoto. Ensemble smoothers for inference of hidden states and parameters in combinatorial regulatory model. Journal of the Franklin Institute, Vol.357(5), pp.2916-2933, 2020. <br/>
	    [20] M. Fujita, R. Yamaguchi, <u>T. Hasegawa</u>, S. Shimada, K, Arihiro, S. Hayashi, K. Maejima, K. Nakano, A. Fujimoto, A. Ono, H. Aikata, M. Uenoh, S. Hayami, H. Tanaka, S. Miyano, H. Yamaue, K. Chayama, K. Kakimi, S. Tanakad, S. Imoto, H. Nakagawa. Classification of primary liver cancer with immunosuppression mechanisms and correlation with genomic alterations. EBioMedicine, Vol.54, p.102737, 2020. <br/>
	    [21] N. Sato, M. Kakuta, <u>T. Hasegawa</u>, R. Yamaguchi, E. Uchino, W. Kobayashi, K. Sawada, Y. Tamura, I. Tokuda, K. Murashita, S. Nakaji, S. Imoto, M. Yanagita and Y. Okuno. Metagenomic analysis of bacterial species in tongue microbiome of current and never smokers. Journal of Oral Microbiology, vol.6(11), pp.11, 2020. <br/>
	    [22,F12] <u>T. Hasegawa</u>, R. Yamaguchi, M. Kakuta, K. Sawada, K. Kawatani, K. Murashita, S. Nakaji and Seiya Imoto.  Prediction of blood test values under different lifestyle scenarios using time-series electronic health record. PLOS ONE, vol.15(3), e0230172, 2020. <br/>
	    [23] A. Fujimoto, M. Fujita, <u>T. Hasegawa</u>, J.H. Wong, K. Maejima, A. Oku-Sasaki, K. Nakano, Y. Shiraishi, S. Miyano, Go Yamamoto, K. Akagi, S. Imoto and H. Nakagawa.Comprehensive analysis of indels in whole-genome microsatellite regions and microsatellite instability across 21 cancer types. Genome Researchm, vol.30, pp.334-346, 2020. <br/>
	    [24,F13] K. Misawa, <u>T. Hasegawa</u>, E. Mishima, P. Jutabha, M. Ouchi, K. Kojima, Y. Kawai, M. Matsuo, N. Anzai, M. Nagasaki. Contribution of Rare Variants of the SLC22A12 Gene to the Missing Heritability of Serum Urate Levels. Genetics, vol.214(4), pp.1079-1090, 2020.<br/>
	    [25] A. Niida, <u>T. Hasegawa</u>, H. Innan, T. Shibata, K. Mimori and S. Miyano. A unified simulation model for understanding the diversity of cancer evolution. PeerJ, vol.8, e8842, 2020. <br/>
	    [26] N.Sato, et al. Metagenomic profiling of gut microbiome in early chronic kidney disease, clinical research, Nephrology Dialysis Transplantation, gfaa172, 2020. <br/>
	    [27,F14] <u>T. Hasegawa</u>, S. Hayashi, E. Shimizu, S. Mizuno, A. Niida, R. Yamaguchim, S. Miyano, H. Nakagawa, and S. Imoto. Neoantimon: A multifunctional R package for identification of tumor-specific neoantigens. Bioinformatics, in press. <br/>
	    [28] K. Itoh, J. Mimura, <u>T. Hasegawa</u>, E. Shimizu, S. Imoto, M. Tsushima, S. Kasai, S. Shimizu, H. Yamazaki, Y Ushida, H. Suganuma, H Tomita, M. Yamamoto and Nakaji. Association of single nucleotide polymorphisms in the NRF2 promoter with vascular stiffness with aging. PLOS ONE, vol.15(8), pp.1-17, 2019.<br/>
	    [29] S. Sakimura, S. Nagayama, M. Fukunaga, Q. Hu, A. Kitagawa, Y. Kobayashi, <u>T. Hasegawa</u>, M. Noda, Y. Kouyama, D. Shimizu, T. Saito, A. Niida, Y. Tsuruda, H. Otsu, Y. Matsumoto, H. Uchida, T. Masuda, K. Sugimachi, S. Sasaki, K. Yamada, K. Takahashi, H. Innan, Y. Suzuki, H. Nakamura, Y. Totoki, S. Mizuno, M. Ohshima, T. Shibata, and K. Mimori. Tumor immune response Determines the Postoperative Recurrence and Detectability of Mutated Genes in ctDNA in Colorectal Cancer Cases, PLOS Genetics, in press. 
	  </p>
	</div>
      </div><!--/.row -->
      <br>
    </div><!--/.container -->
    
    
    <!--SKILLS DESCRIPTION--->
    <div id="skillswrap">
      <div class="container">
		<div class="row">
			<div class="col-lg-2 col-lg-offset-1">
				<h5>Github</h5>
			</div>
			<div class="col-lg-4 centered">
				<canvas id="javascript" height="30" width="30"></canvas>
					<p><a href="https://github.com/hase62"><i class="icon-file"></i></a> <a href="https://github.com/hase62">SSM[3,22],APSKAT[11],Neoantimon[12]</a></p>
				<br>
				<script>
				var doughnutData = [
				{
				value: 70,
				color:"#1abc9c"
				},
				{
				value : 30,
				color : "#ecf0f1"
				}
				];
				var myDoughnut = new Chart(document.getElementById("javascript").getContext("2d")).Doughnut(doughnutData);
				</script>
			</div>
				
			<div class="col-lg-4 centered">
			</div>
		</div>
      </div>
    </div><!--/ #skillswrap -->
    
    <section id="pic" name="pic"></section>
    <!--PORTFOLIO DESCRIPTION -->
    <div class="container desc">
      <div class="row">
		<div class="col-lg-2 col-lg-offset-1">
		<h5>Activity</h5>
		</div>
	
		<div class="col-lg-9">
		<p><img class="img-responsive" src="assets/img/port03.jpg" alt=""></p>
		</div>
	
	  <div class="col-lg-1 col-lg-offset-1">
	  </div>

	  <div class="col-lg-7">
			長谷川嵩矩：東京医科歯科大学 M&Dデータ科学センター 講師 (Junior Associate Professor)。博士(情報学)。<br/><br/>
	
			京都大学大学院 情報学研究科 博士課程の在学中に情報科学技術と芸術の融合を行うべく、CRUNCHERS株式会社を共同創業し、自然言語処理技術を中核とした小説の評価システム(<a href="http://crmg.me/">CRUNCH MAGAZINE</a>)と統計的シミュレーションを用いた映画の興行収益予測システム(CRUNCH CINEMA)を開発(前者は2015年1月に事業譲渡)。その後、博士課程を短縮修了して東北大学東北メディカル・メガバンク機構助教に着任し、疾患と遺伝子変異の関連解析並びに希少変異の網羅的かつ高速な解析手法を開発を行う。<br/><br/>
	
			2015年に東京大学医科学研究所助教へ着任、複雑な性質を持つ時系列データの内部状態とその将来状態を正確に予測する統計理論研究、ICGCプロジェクトにおける免疫ゲノムデータ解析と解析ライブラリ開発、コホートデータを用いたGWASの層別化解析手法の開発や時系列の検査値・生活習慣データとゲノムデータを統合的に利用することで将来の健康状態を予測する統計科学手法の開発などを行う。2020年に現職へ着任。東京大学で行っていた研究に加えて、医科歯科大学内で集められているデータを対象にした生命・健康情報学研究の促進、全学情報ネットワーク改革と生命情報学教育の設立を実施している。省庁などにおける委員会や評価会の委員経験も持ち、臨床医学や臨床検査、医療を含む多くの現場における労働法や派遣法の運用に関する知見も厚い。<br/><br/>
	
			一方で、東京大学の研究を社会へ応用すべく2015年に<a href="http://miup.jp">株式会社miup</a>を共同創業、同時期に<a href="http://medley.life/">Medley, Inc.</a>の技術顧問に着任し、ベイズ学習を用いた臨床診断支援システム(<a href="http://medley.life/symptoms">Mogul</a>)やインフルエンザの感染予測技術などを開発・提供する。その後も、複数のスタートアップ企業やパナソニック株式会社など上場企業の技術・経営顧問を歴任し、いまでは医療・AIやITにこだわらず様々な分野において新規事業の立ち上げ、新規事業戦略の策定、上場企業の経営顧問、経営・システム最適化をコアバリューとして企業再生ファンドの買収部門に対する顧問も行っている。<br/><br/>
	
			<p>Research Bio: Takanori Hasegawa received BS in Engineering from Waseda university, MS in Information Science and Technology from The university of Tokyo, and PhD in Informatics from Kyoto University in 2010, 2012 and 2015, respectively. He is currently an Junior Associate Professor of Integrated Analysis Department, M&D Data Science Center, Tokyo Medical and Dental University. His current research interests cover time series analysis, data assimilation, Bayesian statistical inference, health informatics, genome wide association study, immunological and cancer genome analysis. </p>
		</div>
      </div><!--/.row -->
      <br>
      <br>
    </div><!--/.container -->
  
    <section id="contact" name="contact"></section>
    <!--FOOTER DESCRIPTION -->
    <div id="footwrap">
      <div class="container">
	<div class="row">
	  
	  <div class="col-lg-2 col-lg-offset-1">
	    <h5>CONTACT</h5>
	  </div>
	  <div class="col-lg-5">
	    <p><t>Email</t><br/>
	      t.hasegawa.dsc[at]tmd.ac.jp<br/>
	    </p>
	  </div>
	  <div class="col-lg-4">
	    <p><t>SOCIAL LINKS</t></p>
	    <p>
	      <a href="https://twitter.com/hase62"><i class="icon_booklog"></i></a>
	      <a href="https://www.facebook.com/hase62"><i class="icon-facebook"></i></a>
	      <a href="https://www.linkedin.com/in/hase62"><i class="icon-linkedin"></i></a>
	    </p>
	  </div>
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/ #footer -->
    
    <!--div id="c">
	<div class="container">
	  <p>template<a href="http://www.blacktie.co"></a></p>
	  
	</div>
    </div-->
    
    
    <!-- Bootstrap core JavaScript
	 ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="assets/js/bootstrap.js"></script>
  </body>
</html>
