<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="assets/ico/favicon.png">

    <title>Profile - Takanori Hasegawa, Ph.D</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/css/bootstrap.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="assets/css/main.css" rel="stylesheet">
    
	<link rel="stylesheet" href="assets/css/font-awesome.min.css">
    
    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
    
    <script src="assets/js/jquery.min.js"></script>
    <script type="text/javascript" src="assets/js/smoothscroll.js"></script> 
    <script src="assets/js/Chart.js"></script>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="assets/js/html5shiv.js"></script>
      <script src="assets/js/respond.min.js"></script>
    <![endif]-->
  </head>

  <body data-spy="scroll" data-offset="0" data-target="#nav">
    <div id="section-topbar">
      <div id="topbar-inner">
	<div class="container">
	  <div class="row">
	    <div class="dropdown">
	      <ul id="nav" class="nav">
		<li class="menu-item"><a class="smoothScroll" href="#about" title="About"><i class="icon-user"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#resume" title="Resume"><i class="icon-file"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#current_work" title="Works"><i class="icon-briefcase"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#pub" title="Publications"><i class="icon-pencil"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#pic" title="Pictures"><i class="icon-camera"></i></a></li>
		<li class="menu-item"><a class="smoothScroll" href="#contact" title="Contact"><i class="icon-envelope"></i></a></li>
	      </ul><!--/ uL#nav -->
	    </div><!-- /.dropdown -->
	    
	    <div class="clear"></div>
	  </div><!--/.row -->
	</div><!--/.container -->
	
	<div class="clear"></div>
      </div><!--/ #topbar-inner -->
    </div><!--/ #section-topbar -->
    
    <div id="headerwrap">
      <div class="container">
	<div class="row centered">
	  <div class="col-lg-12">
	    <h1>Takanori Hasegawa, Ph.D</h1>
	    <h3>Assistant Professor at The Univerity of Tokyo</h3>
	    <h3>Co-founder / Collaborative Researcher at miup Inc.</h3>
	    <h3>Technology Advisor at Panasonic Inc. and CareLab Inc.</h3>
	  </div><!--/.col-lg-12 -->
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/.#headerwrap -->    
    
    <section id="about" name="about"></section>
    <div id="intro">
      <div class="container">
	<div class="row">
	  
	  <div class="col-lg-2 col-lg-offset-1">
	    <h5>ABOUT</h5>
	  </div>
	  <div class="col-lg-6">
	    <p>I'm an Assistant Professor at <a href="http://hic-imsut.jp/index.html">The  University of Tokyo</a>, a Co-founder / Collaborative Researcher at <a href="http://miup.jp/">miup Inc.</a>, Technical Advisors at <a href="http://www.panasonic.com">Panasonic, Inc.</a> and <a href="">CareLab Inc.</a> with expertise of Computational Statistics, Time-series Analysis (Data Assimilation), Applied Statistics, Bioinformatics, and Machine Learning. I'm interested in applying technological developments in computer science and biotechnology for the innovation of human being. 

In basic research for Statistical Science, I have developed several types of data assimilation methodologies, e.g., linear/nonlinear Kalman filters. In practical research, I have analyzed human health and epidemiological data such as time-series medical examination data and cancer genome sequencing to predict future health risk and establish personalized medicine, to elucidate immune cancer relationships, and to prevent disease pandemic. In business field, the current interest is to develop several types of artificial intelligence systems utilizing, e.g., Bayesian statistical inference, for the application of medicine, health and nursing care, and immune-cancer therapies. I'm also studying modern history, classical strategy, e.g., Hundred Schools of Thought and The Prince, geopolitics and philosophy as life works.</p>
	  </div>
	  <!--div class="col-lg-3">
	      <p><a href="#"><i class="icon-file"></i></a> <sm>DOWNLOAD PDF</sm></p>
	  </div!--->
	  
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/ #intro -->

    
    <section id="resume" name="resume"></section>
    <!--EDUCATION DESCRIPTION -->
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>EDUCATION</h5>
	</div>
	<div class="col-lg-6">
	  <p><t>Doctor of Informatics</t><br/>
	    Department of  Intelligence Science and Technology<br/>
	    Graduate School of Informatics, Kyoto University<br/>
	    (Bioinformatics Center, Institute for Chemical Research, Kyoto University)<br/>
	    <br/>
	    <i>Doctoral Program</i><br/>
	    Thesis Title: Reconstructing Biological Systems Incorporating Multi-Source Biological Data via Data Assimilation Techniques<br/>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2012 to Jan. 2015 (Completion in shortened years)</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Master of Information Science and Technology</t><br/>
	    Department of Computer Science <br/>
	    Graduate School of Information Science, The University of Tokyo<br/>
	    (Human Genome Center, Institute of Medical Science, The University of Tokyo)<br/>
	    <br/>
	    <i>Master's Program</i><br/>
	    Thesis Title: Intracellular Systems Analysis by Assimilating Large Scale Biological Data and Pathway Simulation<br/>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2010 to Mar. 2012</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Bachelor of Engineering</t><br/>
	    Facalty of Science and Engineering, Waseda University<br/>
	    <br/>
	    <i>Baccalaureate Degree Program</i><br/>
	    Thesis Title: 遺伝子ネットワーク推定とデータ同化による細胞内ネットワークシミュレーションモデルの構築</br>
	  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>From Apr. 2006 to Mar. 2010</sm></p>
	</div>
	
      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->
    
    
    <!--WORK DESCRIPTION -->
    <section id="current_work" name="current_work"></section>
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>CURRENT WORK</h5>
	</div>
		
	<div class="col-lg-6">
	  <p><t>Assistant Professor</t><br/>
	    <a href="http://www.ims.u-tokyo.ac.jp/imsut/jp/lab/healthintelligence/">Health Intelligence Center, The Institute of Medical Science, The University of Tokyo</a><br/>
	  </p>
	  <p>Our research topics are human genome, transcriptome, epigenome and clinical data for personalized and preventive medicine through modeling, prediction and inference of disease and health systems. For example, we have partially focused on immunological cancer genome and metagenome analysis based on statistical science for these purposes. On the other hand, I'm also focusing on the theoretical developments in data assimilation. </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>October 2015 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Co-founder / Collaborative Researcher</t><br/>
	    <a href="http://miup.jp/">miup</a><br/>
	  </p>
	  <p>
	    We're trying to establish "Integrated Healthcare Eco-System" using IoT technology and clinical A.I. system for developing countries. Our system can triage and assess one's health risk without professional knowledge given symptoms and/or test values, suggest a candidate list of treatments, and support clinical decision for medical doctors. The system has a highly complicated statistical structure to enhance the quality of the inference, estimate the confidence level of the conclusions, and predict the future risks through Bayesian statistics and machine learning (We have no plan to publish the detailed statistical structures). If you're interested in the program, please send me a message. </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>September  2015 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Technology Advisor</t><br/>
	    <a href="http://nadia.bz/en/">Panasonic Inc.</a><br/>
	  </p>
	  <p>Confidential: A.I. Development</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Apr 2017 - Current</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Collaborative Project / Technology Advisor</t><br/>
	    <a href="http://nadia.bz/en/">CareLab Inc.</a><br/>
	  </p>
	  <p>Confidential: A.I. Development</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Apr 2017 - Current</sm></p>
	</div>

      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->

    <!--WORK DESCRIPTION -->
    <section id="past_work" name="past_work"></section>
    <div class="container desc">
      <div class="row">
	
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>PAST WORK</h5>
	</div>

	<div class="col-lg-6">
	  <p><t>Technology Advisor</t><br/>
	    <a href="http://medley.life/">Medley, Inc.</a><br/>
	  </p>
	  <p>We developed an automated consulting system termed <a href="http://medley.life/symptoms">Mogul</a> for medical treatment. This system is a type of artificial intelligence and based on a simple Bayesian inference procedure (Its concept is similar to the one in miup Inc. but this version is a little bit older). Also, a pandemic prediction system for the degree of future influenza epidemic has been released in the website. </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>February  2015 - April 2017</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Technology Advisor</t><br/>
	    <a href="http://nadia.bz/en/">Nadia Inc.</a><br/>
	  </p>
	  <p>Confidential: A.I. Development</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Mar 2016 - Feb 2017</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Technology Advisor</t><br/>
	    <a href="http://www.healthcareonline.co.jp/">Heathcare online</a><br/>
	  </p>
	  <p>Confidential: A.I. Development</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>Mar 2016 - Aug 2016</sm></p>
	</div>

	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Assistant Professor</t><br/>
	    <a href="http://www.megabank.tohoku.ac.jp/english/">Tohoku Medical Megaban Organization, Tohoku University</a><br/>
	  </p>
	  <p>The main topic was genome wide association study and to develop novel strategies for analyzing rare variants association test. Our developed method is highly efficient and available to WGS data within a reasonable time. In addition, we proposed a novel statistical method termed kernel Bayes' approximate Bayesian computation filtering for estimation of parameter values in stochastic simulation models.  </p>
	</div>
	<div class="col-lg-3">
	  <p><sm>February  2015 - September 2015</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Co-founder / CTO (Technology Advisor, Feb. 2015-)</t><br/>
	    <a href="http://www.crunchers.jp/">CRUNCHERS Inc. </a><br/>
	  </p>
	  <p>We developed automated evaluation systems for predicting the profit of books and cinema. The former system was integrated to Medley, Inc. as <a href="http://crmg.me/">CRUNCH MAGAZINE</a>. For the latter system, I still have advised the statistical perspective of applying the developed simulation system (CRUNCH CINEMA) to the collaborative companies. In addition, since we're collaborating with major studios and major entertainment companies in JAPAN and USA, we have extended and developed CRUNCH CINEMA for the application of general purpose.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>July 2013 - September 2015</sm></p>
	</div>
	
	<div class="col-lg-6 col-lg-offset-3">
	  <p><t>Research Fellow</t><br/>
	    Japan Society for The Promotion of Science<br/>
	  </p>
	  <p>The research topic was Reconstructing Biological Systems Incorporating Multi-Source Biological Data via Data Assimilation Techniques.</p>
	</div>
	<div class="col-lg-3">
	  <p><sm>April 2012 - January 2015</sm></p>
	</div>
	
      </div><!--/.row -->
      <br>
      <hr>
    </div><!--/.container -->
    
    
    <!--AWARDS DESCRIPTION -->
    <section id="pub" name="pub"></section>
    <div class="container desc">
      <div class="row">
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>PUBLICATIONS</h5>
	</div>
	<div class="col-lg-9">
	  <p><t>Journal Paper</t><br/>
	    [J1] <u>T. Hasegawa</u>, M. Nagasaki, R. Yamaguchi, S. Imoto and S. Miyano. An efficient method of exploring simulation models by assimilating literature and biological observational data, BioSystems, vol.121, pp.54-66, 2014.<br/>
	    [J2] <u>T. Hasegawa</u>, R. Yamaguchi, M. Nagasaki, S. Miyano and S. Imoto. Inference of gene regulatory networks incorporating multi-Source biological knowledge via a state space model with L1 regularization, PLoS ONE, vol.9(8), e105942, 2014.<br/> 
	    [J3] <u>T. Hasegawa</u>, T. Mori, R. Yamaguchi, S. Imoto, S. Miyano and T. Akutsu. An efficient data assimilation schema for restoration and extension of gene regulatory networks using time-course observation data, Journal of Computational Biology, vol.21(11), pp. 785-798, 2014.<br/>
	    [J4] <u>T. Hasegawa</u>, T. Mori, R. Yamaguchi, T. Shimamura, S. Miyano, S. Imoto and T. Akutsu. Genomic data assimilation using a higher moment filtering technique for restoration of gene regulatory networks. BMC Systems Biology, vol.9(14), 2015.<br/>
	    [J5] <u>T. Hasegawa</u>, A. Niida, T. Mori, T. Shimamura, R. Yamaguchi, S. Miyano, T. Akutsu and S. Imoto. A likelihood-free filtering method via approximate Bayesian computation in evaluating biological simulation models. Computational Statistics and Data Analysis, vol.94, pp.63-74, 2016.<br/>
	    [J6] J. Kurashige, <u>T. Hasegawa (Equal First)</u>, et al. Integrated molecular profiling of human gastric cancer identifies DDR2 as a potential regulator of peritoneal dissemination. Scientific Reports, vol.6, pp.22371, 2016.<br/>
	    [J7] E. Ayada, <u>T. Hasegawa</u>, A. Niida, S. Miyano and S. Imoto. Binary contingency table Method for analyzing gene mutation in cancer genome. International Journal of Bioinformatics Research and Applications (11th The International Symposium on Bioinformatics Research and Applications), vol.12(3), pp.211-226, 2016.<br/>
	    [J8] K. Kojima, Y. Kawai, N. Nariai, T. Mimori, <u>T. Hasegawa</u> and M. Nagasaki. Short tandem repeat number estimation from paired-end sequence reads by considering unobserved genealogy of multiple individuals. BMC Genomics (11th The International Symposium on Bioinformatics Research and Applications), vol.17(Suppl. 5), pp.465--476, 2016.<br/>
	    [J9] S. Koshiba, Y. Yamaguchi, K. Kojima, <u>T. Hasegawa</u>, et al. The structural origin of metabolic quantitative diversity. Scientific Reports, vol.6, pp.31463, 2016.<br/>
	    [J10] <u>T. Hasegawa</u>, K. Kojima, Y. Kawai, K. Misawa, T. Mimori and M. Nagasaki. AP-SKAT: highly-efficient genome-wide rare variant association test, BMC Genomics, vol.17(1), pp.1-8, 2016.
	    [J11] T. Morita, A. Rahman, <u>T. Hasegawa</u>, A. Ozaki, Tetsuya Tanimoto. The Potential Possibility of Symptom Checker, International Journal of Health Policy and Management, 2017. 
	  </p>
	  <p><t>Peer-reviewed Oral Presentation with Proceeding/Lecture Note (Not Published in Journals)</t><br/>
	    [C1] <u>T. Hasegawa</u>, R. Yamaguchi, M. Nagasaki, S. Imoto and S. Miyano. Poster: Comprehensive pharmacogenomic pathway screening by data assimilation, Proc. Computational Advances in Bio and Medical Sciences (ICCABS), 2011 IEEE 1st International Conference on, pp.246-246, 2011.<br/>
	    [C2] <u>T. Hasegawa</u>, R. Yamaguchi, M. Nagasaki, S. Imoto and S. Miyano. Comprehensive pharmacogenomic pathway screening by data assimilation, Proc. 7th International Symposium on Bioinformatics Research and Applications, volume 6674 of Lecture Notes in Computer Science, pp.160-171. Springer Berlin Heidelberg, 2011.<br/>
	    [C3] <u>T. Hasegawa</u>, S. Hayashi, E. Shimizu, S. Mizuno, R. Yamaguchi, S. Miyano, H. Nakagawa, and S. Imoto. An in silico automated pipeline to identify tumor specic neoantigens from whole genome and exome sequencing data. Proc. 12th International Symposium on Bioinformatics Research and Applications, 2016.<br/>
	  </p>
	</div>
      </div><!--/.row -->
      <br>
    </div><!--/.container -->
    
    
    <!--SKILLS DESCRIPTION--->
    <div id="skillswrap">
      <div class="container">
	<div class="row">
	  <div class="col-lg-2 col-lg-offset-1">
	    <h5>Downloads</h5>
	  </div>
	  <div class="col-lg-4 centered">
	    <canvas id="javascript" height="50" width="50"></canvas>
            <p><a href="#"><i class="icon-file"></i></a> <a href="#">VARSSM.jar[J2]</a></p>
	    <br>
	    <script>
	      var doughnutData = [
	      {
	      value: 70,
	      color:"#1abc9c"
	      },
	      {
	      value : 30,
	      color : "#ecf0f1"
	      }
	      ];
	      var myDoughnut = new Chart(document.getElementById("javascript").getContext("2d")).Doughnut(doughnutData);
	    </script>
	  </div>
	  <div class="col-lg-4 centered">
	    <canvas id="bootstrap" height="50" width="50"></canvas>
	    <p><a href="#"><i class="icon-file"></i></a> <a href="#">HMEnPF.jar[J4]</a></p>
	    <br>
	    <script>
	      var doughnutData = [
	      {
	      value: 90,
	      color:"#1abc9c"
	      },
	      {
	      value : 10,
	      color : "#ecf0f1"
	      }
	      ];
	      var myDoughnut = new Chart(document.getElementById("bootstrap").getContext("2d")).Doughnut(doughnutData);
	    </script>
	  </div>	  
	  <div class="col-lg-4 centered">
	    <canvas id="bootstrap" height="50" width="50"></canvas>
	    <p><a href="https://github.com/hase62/APSKAT"><i class="icon-file"></i></a> <a href="https://github.com/hase62/APSKAT">APSKAT[J10]</a></p>
	    <br>
	    <script>
	      var doughnutData = [
	      {
	      value: 40,
	      color:"#1abc9c"
	      },
	      {
	      value : 60,
	      color : "#ecf0f1"
	      }
	      ];
	      var myDoughnut = new Chart(document.getElementById("bootstrap").getContext("2d")).Doughnut(doughnutData);
	    </script>
	  </div>
	  <div class="col-lg-4 centered">
	    <canvas id="bootstrap" height="50" width="50"></canvas>
	    <p><a href="https://github.com/hase62/Neoantimon"><i class="icon-file"></i></a> <a href="https://github.com/hase62/Neoantimon">Neoantimon[C3]</a></p>
	    <br>
	    <script>
	      var doughnutData = [
	      {
	      value: 100,
	      color:"#1abc9c"
	      },
	      {
	      value : 0,
	      color : "#ecf0f1"
	      }
	      ];
	      var myDoughnut = new Chart(document.getElementById("bootstrap").getContext("2d")).Doughnut(doughnutData);
	    </script>
	  </div>
	</div>
	<br>
      </div>
    </div><!--/ #skillswrap -->
    
    <section id="pic" name="pic"></section>
    <!--PORTFOLIO DESCRIPTION -->
    <div class="container desc">
      <div class="row">
	<div class="col-lg-2 col-lg-offset-1">
	  <h5>Activity</h5>
	</div>
	
	<div class="col-lg-9">
	  <p><img class="img-responsive" src="assets/img/port01.jpg" alt=""></p>
	</div>
	
	<div class="col-lg-9">
	  <p>東京大学医科学研究所 ヘルスインテリジェンスセンター 助教。博士(情報学)。修士・博士課程に於いて情報科学を専攻、生命情報学研究（生体内システムの統計的解析手法開発）に携わる。2013年に情報科学技術と芸術の融合を行うべく、CRUNCHERS株式会社を共同創業し、自然言語処理技術を中核とした小説の評価システム(<a href="http://crmg.me/">CRUNCH MAGAZINE</a>)と統計的シミュレーションを用いた映画の興行収益予測システム(CRUNCH CINEMA)を開発(前者は2015年1月に事業譲渡)。京都大学大学院 情報学研究科博士課程を短縮修了後、東北大学東北メディカル・メガバンク機構助教に着任し、疾患と遺伝子変異の関連解析並びに希少変異の網羅的かつ高速な解析手法を開発を行う。また同時期から<a href="http://medley.life/">Medley, Inc.</a>の技術顧問に着任し、ベイズ学習を用いた医療用疾患推定システム(<a href="http://medley.life/symptoms">Mogul</a>)やインフルエンザの感染予測技術などを開発・提供する。現職では、複雑な性質を持つ時系列データの内部状態とその将来状態を正確に予測する統計理論研究、ICGCプロジェクトのメンバーとしての免疫ゲノムデータ解析、コホートスタディデータを用いた疾患の将来リスク予測モデル開発の他、症状や検査値から疾患の予知・予防を行う統計科学手法の開発・応用を行っている。そのほか複数企業のAI(統計科学、機械学習)開発の技術顧問をする。生命情報学、統計科学、統計的シミュレーションが専門であり、特に状態空間モデルの理論的進展に関する幅広い知見を有している。<br/><br/>
	  </p>
	  <p>Research Bio: Takanori Hasegawa is currently an Assistant Professor of Health Intelligence Center, Institute of Medical Science, University of Tokyo. He received BS in Engineering from Waseda university, MS in Information Science and Technology from The university of Tokyo, and PhD in Informatics from Kyoto University in 2010, 2012 and 2015, respectively. His current research interests cover time series analysis using state space models, e.g., genomic data assimilation, Bayesian statistical inference to assess one's helath risks and help M.D. to diagnose one's diseases, immunological and cancer genome analysis. </p>
	</div>
	
	<!---div class="col-lg-6 col-lg-offset-3">
	    <p><img class="img-responsive" src="assets/img/port03.jpg" alt=""></p>
      </div>
      <div class="col-lg-3">
	<p>WE GROW</p>
	<p><more>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.<br/><br/>
	    <sm><i class="icon-tag"></i> web design</sm></more> 
	</p>
	</div--->
	
      </div><!--/.row -->
      <br>
      <br>
    </div><!--/.container -->
    
    
    
    <section id="contact" name="contact"></section>
    <!--FOOTER DESCRIPTION -->
    <div id="footwrap">
      <div class="container">
	<div class="row">
	  
	  <div class="col-lg-2 col-lg-offset-1">
	    <h5>CONTACT</h5>
	  </div>
	  <div class="col-lg-6">
	    <p><t>Email</t><br/>
	      t-hasegw@ims.u-tokyo.ac.jp<br/>
	    </p>
	  </div>
	  <div class="col-lg-3">
	    <p><sm>SOCIAL LINKS</sm></p>
	    <p>
	      <!---a href="https://twitter.com/hase62"><i class="icon-twitter"></i></a!--->
	      <a href="https://www.facebook.com/hase62"><i class="icon-facebook"></i></a>
	      <a href="https://www.linkedin.com/in/hase62"><i class="icon-linkedin"></i></a>
	      <a href="https://www.wantedly.com/users/142648"><i class="glyphicon glyphicon-briefcase"></i></a>
	      <a href="http://coffeemeeting.jp/1352542156"><i class="icon-coffee"></i></a>
	    </p>
	  </div>
	</div><!--/.row -->
      </div><!--/.container -->
    </div><!--/ #footer -->
    
    <!--div id="c">
	<div class="container">
	  <p>template<a href="http://www.blacktie.co"></a></p>
	  
	</div>
    </div-->
    
    
    <!-- Bootstrap core JavaScript
	 ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="assets/js/bootstrap.js"></script>
  </body>
</html>
